packages/runtime/src/next/components/tests/controls/color-control/__snapshots__/client.test.tsx.snap

```diff
@@ -1,8 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Page Color control data v0 with value: {"alpha": 0.5, "swatchId": "[swatch-test-id]"}, swatch: {"__typename": "Swatch", "hue": 238, "lightness": 49, "saturation": 87}, default value: rgb(255,0,0) fetching swatch: resolvedValue 1`] = `"rgba(16, 24, 234, 0.5)"`;
+exports[`color data v0 with swatch {"__typename": "Swatch", "hue": 238, "lightness": 49, "saturation": 87} fetching swatch: resolvedValue 1`] = `"rgba(16, 24, 234, 0.5)"`;
```

Comment:
Cleaned up/simplified existing snapshots' titles. No changes to snapshot values.